aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -583,7 +603,18 @@ struct algorithm_search<cudnnConvolutionBwdDataAlgoPerf_t> {
           ws.data,
           ws.size));
     }
-    return getBestAlgorithm<perf_t>(perf_results.get(), args.params.deterministic, perf_count);
+    
+    // See Note [blacklist fft algorithms for strided dgrad]
+    int stride_dim = args.input.dim() - 2;
+    bool blacklist = false;
+    for (int i = 0; i< stride_dim; i++) {
```

Comment:
nit: this can be `bool blacklist = std::equal(args.params.stride, args.params.stride + stride_dim, 1)`